pkgs/by-name/ca/cargo-gg/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  rustPlatform,
+  fetchCrate,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-gg";
+  version = "0.1.1";
+
+  src = fetchCrate {
+    inherit version;
+    pname = "cargo-gg";
+    hash = "sha256-O8RwD3M7MGLyTIRqWRfKoDmaaT+ylxM4e4PcayyrA+c=";
+  };
+
```

Comment:
fetchFromGitLab or fetchGit is preferred

https://gitlab.scd31.com/stephen/cargo-gg

Reply:
Updated!